tests/specs/parse-tsconfig/parses.spec.ts

```diff
@@ -76,7 +76,7 @@ export default testSuite(({ describe }) => {
 				'file.ts': '',
 				'tsconfig.json': tsconfigJsonString({
 					compilerOptions: {
-						moduleResolution: 'node',
+						moduleResolution: 'node10',
```

Comment:
Why?

Reply:
It seems that the option value has changed from `node` to `node10` in TypeScript 5.0. The internal name has been changed accordingly.

https://github.com/microsoft/TypeScript/pull/51901

Reply:
Say huh

Reply:
If it's mine it's only because I switched from Apple to android?  How ever I did receive a call marked as Unknown!  From a co worked who claimed no knowledge but I still have the message??  It was sent in a group